addons/hr_skills/security/hr_skills_security.xml

```diff
@@ -51,6 +51,21 @@
         <field name="groups" eval="[(4,ref('base.group_user'))]"/>
     </record>
 
+    <record id="hr_employee_skill_log_hr_user_rule" model="ir.rule">
+        <field name="name">Employee Skill History:  : read</field>
+        <field name="model_id" ref="model_hr_employee_skill_log"/>
+        <field name="domain_force">[(1, '=', 1)]</field>
+        <field name="groups" eval="[(4, ref('hr.group_hr_user'))]"/>
+    </record>
+
+    <record id="hr_employee_skill_log_manager_rule" model="ir.rule">
+        <field name="name">Employee Skill History: employee's manager : read</field>
+        <field name="model_id" ref="model_hr_employee_skill_log"/>
+        <!-- <field name="domain_force">[(0, '=', 1)]</field> -->
+        <field name="domain_force">['|', ('employee_id.id', 'in', user.employee_ids.subordinate_ids.ids), ('employee_id.id', 'in', user.employee_ids.ids)]</field>
```

Comment:
hello, i think we can achieve the same result using the child_of operator like this
`        <field name="domain_force">[('employee_id.id', 'child_of', user.employee_ids.ids)]</field>
`
without moving the code from org chart, but has to be tested first

Reply:
but you will just see your direct subordinate not your indirect subordinate no ?

Reply:
no, with child off, you will see all the subordinates :bulb:

Reply:
ok i will change that :ok_hand: